action.yml

```diff
@@ -9,7 +9,7 @@ inputs:
     description: |
       Path to the directory containing the manifest_file.
     required: false
-    default: ${{ github.workspace }}
+    default: Working directory
```

Comment:
I think this should be `.` instead of `working directory`
because this default value is considered when input is not provided, so in case of no input `manifest_dir` it will be equal to `working directory` which is wrong.